### DISCUSSION TITLE
2023/09/23/stack-and-heap

### DISCUSSION THREAD

**Discussion by giscus** [kaochenlong/kaochenlong-discuss#8] [General]
# 2023/09/23/stack-and-heap

這個章節跟 Rust 比較沒直接關連，比較像是計算機概論的科普時間。這個章節我們要跟大家介紹關於記憶體的 Stack 與 Heap。通常 Stack 中文會翻譯成「堆疊」，而 Heap 會翻譯成「堆積」，但我覺得中文翻譯好像沒比較容易懂，所以我們這裡就用英文原文就好。在我學習技術的這條路上，當我想要了解一個新的名...

https://kaochenlong.com/2023/09/23/stack-and-heap.html

**Comment by whdinasor** [comment]
謝謝,這是我看到最清楚的解釋了...
